health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/AddDeliverycontext.js

```
@@ -590,62 +640,33 @@ const AddDeliveryRule = ({ targetedData, deliveryRules, setDeliveryRules, index,
   );
 };
 
-const AddDeliveryRuleWrapper = ({ }) => {
+const AddDeliveryRuleWrapper = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)